<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelicalServerDetailsDialog</class>
 <widget class="QDialog" name="HelicalServerDetailsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>49</y>
     <width>351</width>
     <height>213</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="connectionNameLineEdit"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="serverPortLineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="commandLabel">
      <property name="text">
       <string>Command:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="commandLineEdit"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="userPasswordLineEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="serverNameLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="serverPortLabel">
      <property name="text">
       <string>Port:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="connectionNameLabel">
      <property name="text">
       <string>Connection Name:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="serverNameLabel">
      <property name="text">
       <string>Server:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="userNameLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="userPasswordLabel">
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="userNameLabel">
      <property name="text">
       <string>User:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="userHomeLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="userHomeLabel">
      <property name="text">
       <string>Home:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget1">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>168</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
